drivers/scheduler/sidecars/es-stress-test/requirements.txt

```diff
@@ -0,0 +1 @@
+elasticsearch
```

Comment:
I guess this is better to put this inside dockerfile

Reply:
This file is passed as docker build context and a base image use it to download dependencies.

Reply:
this is part of https://github.com/logzio/elasticsearch-stress-test/blob/master/requirements.txt as well, let's keep it simple

Reply:
done